src/net/SDP/SDPMessageMediaFormat.cs

```diff
@@ -0,0 +1,31 @@
+﻿//-----------------------------------------------------------------------------
+// Filename: SDPMessageMediaFormat.cs
+//
+// Description: Contains enums and helper classes for common definitions
+// and attributes used in SDP payloads.
+//
+// Author(s):
+// Jacek Dzija
+// Mateusz Greczek
+//
+// History:
+// 30 Mar 2021 Jacek Dzija,Mateusz Greczek Added MSRP
+//
+// License: 
+// BSD 3-Clause "New" or "Revised" License, see included LICENSE.md file.
+//-----------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace SIPSorcery.net.SDP
+{
+    public class SDPMessageMediaFormat
+    {
+        public List<string> AcceptTypes;
+
+        public string Endpoint;
+        public string IP;
+
+        public string Port;
+    }
+}
```

Comment:
Does this class correspond to these MSRP SDP attributes? What does the `IP` field represent? Does there need to another field to represent the transport type such as `tcp`?

````
a=accept-types:text/plain
a=path:msrp://biloxi.example.com:12763/kjhd37s2s20w2a;tcp
````

Reply:
port and IP are needed for incoming SDS offer since it could be hosted in another server